steps/hearing/arrangements/template.html

```diff
@@ -10,7 +10,7 @@
 {% block fields %}
 
 <p>{{ content.clickAllArrangements | safe }}</p>
-<p>{{ content.cannotBringInterpreter | safe }}</p>
+<p>{{ content.you }}<strong class="bold-small"> {{ content.cant }}</strong> {{ content.bringYourOwnInterpreter }}</p>
```

Comment:
Couldn't you put the html as one thing in the content? Or do you want to keep the content and html separate?

Reply:
Yeah, I looked at that. We really shouldn't be mixing HTML and content. I know we have it elsewhere but I'm concerned that when these files get translated to Welsh we will have problems.